keep/README.md

```diff
@@ -0,0 +1,42 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
+## Setup
+
+### Installation
```

Comment:
As mentioned in this [article](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/#publish-the-oauth-client), please include uninstallation steps.